src/current/_includes/v23.1/cdc/sink-configuration-detail.md

```diff
@@ -0,0 +1,28 @@
+{{site.data.alerts.callout_danger}}
+Setting either `Messages` or `Bytes` with a non-zero value without setting `Frequency`, will cause the sink to assume `Frequency` has an infinity value. If either `Messages` or `Bytes` have a non-zero value, then a non-zero value for `Frequency` **must** be provided. This configuration is invalid and will cause an error, since the messages could sit in a batch indefinitely if the other conditions do not trigger.
+{{site.data.alerts.end}}
+
+Some complexities to consider when setting `Flush` fields for batching:
+
+- When all batching parameters are zero (`"Messages"`, `"Bytes"`, and `"Frequency"`) the sink will interpret this configuration as "send batch every time." This would be the same as not providing any configuration at all:
```

Comment:
nit: change "every time" to "every time a message is available"

Reply:
Done!